ClientApp/e2e/pageobject/pages/candidateview/candidateview.po.ts

```diff
@@ -1,27 +1,27 @@
-import { CommonPageHelper } from '../common/common-page.helper';
+import { CommonPageHelper } from '../common/CommonPageHelper';
 
 let locator = require("../property/locator");
-let helper: CommonPageHelper;
 
-export class CandidateView {
-
-    helper = new CommonPageHelper();
-
-    isPresent(elementName:string){
-     helper.isPresent(locator.details[elementName]);
+export class CandidateView extends CommonPageHelper {
+   constructor() {
+      super(locator);
     }
 
-   getText(elementName:string){
-      return helper.getText(locator.details[elementName]);
+    setCandidateInfo(data){
+         this.sendKeys(locator.editFirstName, data.firstName);
```

Comment:
As we discussed we must have a function that will check the element presence before you set the value. Can't see that here?

Reply:
Its ok got it :)